src/components/Questions/Question.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
same thing with others.. dont you think its better to change the name of Question.js to something else such as AdminFromQuestions.js